<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminCourseDecision</class>
 <widget class="QDialog" name="adminCourseDecision">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:light gray</string>
  </property>
  <widget class="QLabel" name="adminmModeClassLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>421</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Administrator Course Modify</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="facultyNameEntry">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="addConfirmCourse">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Course</string>
   </property>
  </widget>
  <widget class="QLabel" name="prerequisitesLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Course Prerequisites (List Using &quot;,&quot;):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="courseNameEntry">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="coursePrerequisitesEntry">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="courseLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Course Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorMessageBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>491</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="removeConfirmCourse">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remove Course</string>
   </property>
  </widget>
  <widget class="QLabel" name="facultyLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>61</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Faculty:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
